api/tests/routes/native/v1/bookings_test.py

```diff
@@ -897,7 +898,8 @@ def test_cancel_booking(self, client):
         booking = booking_factories.BookingFactory(user=user)
 
         client = client.with_token(self.identifier)
-        response = client.post(f"/native/v1/bookings/{booking.id}/cancel")
+        with assert_num_queries(29):
```

Comment:
Oo

Reply:
❤️  le commit par contre

Reply:
Ouais j'ai pas osé mettre le détail, mais j'ai noté de faire une passe là-dessus en journée back ou autre.